frontend/src/components/LandingPage.js

```diff
@@ -4,57 +4,64 @@ import { Link } from "react-router-dom";
 import Header from "./Header";
 import BalancePresentation from "./BalancePresentation";
 import Footer from "./Footer";
+import "../styles/LandingPage.css";
 
 function Landing() {
   return (
-    <>
+    <div className="LandingPage">
       <Header />
       <main>
         <BalancePresentation />
         <Accordion>
           <Card>
             <Accordion.Toggle as={Card.Header} eventKey="0">
-              Update salary
+              <Button className="w-100 text-white">Update Salary</Button>
             </Accordion.Toggle>
             <Accordion.Collapse eventKey="0">
               <Card.Body>
                 <Form>
-                  <Form.Group controlId="formSalary">
-                    <Form.Label>Salary</Form.Label>
-                    <Form.Control type="number" /> €
-                  </Form.Group>
-                  <Button variant="primary">Save</Button>
+                  <div className="form-wrapper">
```

Comment:
This also is polluting the dom with unnecessary wrapper divs. The styles could be applied directly to the Form component.

Reply:
changes done!

Reply:
Resolved in 5f37977c717e71dfe2cbcf56d4668acac47e6947